### ISSUE TITLE
A question about setting f=f.detach() in ft_netAB

### ISSUE THREAD

**Issue by RainbowShirlley** [NVlabs/DG-Net#59]
Hi, 
thank you for your code. 
I have a question about the code 'f=f.detach()' in ft_netAB of reIDmodel.py. 
Since f set to to be detached, then loss(4) in your paper will have two appearance codes which are both inable to backward，thus the loss with be useless. Moreover, since f is used to reconstruct images, then losses for image reconstruction will also have no effect for Ea's updating.
I'm so confused, thank you for your kind explanation!

**Comment by RainbowShirlley** [comment]
Hello,

**Comment by RainbowShirlley** [comment]
Hello, can anybody help me with the question above? I'm really confused, and hope for rational explanation, thanks!
